### PR TITLE
Team page revamp

### PR DESCRIPTION
**Pull request by cherchezlafemme**
Team page update: 
- removes past contributors descriptions, moves them into dedicated section;
- adds new team members, updates pictures;
- simplifies css (removes some duplicate css, replace to use standard bootstrap classes);
- attempt to fix markup in the readme https://github.com/shescoding/shescoding-dot-org/blob/7f3db634fb39851fc6765e18f72b1173bcb31e3f/README.md in this PR. the indentation was off.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -82,3 +82,7 @@ body {
 img {
   max-width: 100%;
 }
+
+.title {
+  font-size: 36px;
```
Comment (natsteinmetz) [inline app/assets/stylesheets/application.scss]: Add top padding of 20px to keep more space above the titles.

File: app/assets/stylesheets/team.css.scss [hunk 2]
```diff
@@ -2,15 +2,14 @@
...
-.team-member {
-    padding-top: 30px;
+$black: #1E2028;
+$red: #DB2C38;
+$green: #83C057;
+$blue: #00A0BE;
+$past_cont_grey: #808080;
+
+.team-page-section {
+    padding: 50px 0;
```
Comment (natsteinmetz) [inline app/assets/stylesheets/team.css.scss]: This one will need to be reduced to make up for the general application title padding.

Comment (cherchezlafemme) [reply app/assets/stylesheets/team.css.scss]: Updated

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (cherchezlafemme) [reply app/assets/stylesheets/application.scss]: yeah, make sense! Added padding.

**Pull request merged**